apps/angular/src/app/features/anime-filter-form/anime-filter-form.component.ts

```diff
@@ -29,57 +30,52 @@ import { AnimeFilterForm } from './anime-filter-form.model';
 		MatSelectModule,
 	],
 })
-export class AnimeFilterFormComponent implements OnInit, OnDestroy {
-
-	/** Form group for anime filter form. */
-	public animeFilterFormGroup?: FormGroup<AnimeFilterForm>;
+export class AnimeFilterFormComponent implements OnInit {
 
 	/** Initial value for search control. */
 	@Input() public searchValue = '';
 
-	/** Event of search form submitting. */
-	@Output() public searchEvent = new EventEmitter<string>();
-
 	/** Initial value for anime types control. */
 	@Input() public typesValue: AnimeType[] = [];
 
-	/** Event of anime type selection. */
-	@Output() public typeSelectEvent = new EventEmitter<AnimeType[]>();
+	/** Event of anime types or search term changes. */
+	@Output() public animeFiltersEvent = new EventEmitter<Partial<AnimeFilters>>();
 
-	/** List of permissible values for types select control. */
-	protected typesList = Object.values(AnimeType);
+	/** Form group for anime filter form. */
+	protected readonly animeFilterFormGroup: FormGroup<AnimeFilterForm>;
 
-	private searchChangesSubscription?: Subscription;
+	/** List of permissible values for types select control. */
+	protected readonly typesList = Object.values(AnimeType);
```

Comment:
You don't need to add `List` if that word is already plural and the naming can improve a bit 
```suggestion
	protected readonly animeTypes = Object.values(AnimeType);
```